### ISSUE TITLE
save and load

### ISSUE THREAD

**Issue by lorepieri8** [ibarrond/Pyfhel#40]
```
from Pyfhel import Pyfhel, PyPtxt, PyCtxt

def save_and_load():

	HE = Pyfhel()
	HE.contextGen(p=65537)
	HE.keyGen()
	integer1 = 10
	ctxt = HE.encryptInt(integer1)
	ctxt.save("ctxt.c1")
	m = PyCtxt()
	result = m.load("ctxt.c1")
	print(result)
	print(HE.decryptInt(result))


if __name__ == "__main__":
	save_and_load()
	print('------------')
```

Output:

```
------------
None
Traceback (most recent call last):
  File "Pyfhel/Pyfhel.pyx", line 338, in Pyfhel.Pyfhel.Pyfhel.decryptInt
RuntimeError: <Pyfhel ERROR> wrong encoding type in PyCtxt

The above exception was the direct cause of the following exception:

[...]
SystemError: <method 'decryptInt' of 'Pyfhel.Pyfhel.Pyfhel' objects> returned a result with an error set

```

**Comment by lorepieri8** [comment]
Related to #33 .

**Comment by lorepieri8** [comment]
The following code correctly saving-loading:

```
HE = Pyfhel()
	HE.contextGen(p=65537)
	HE.keyGen()
	integer1 = 10
	ctxt = HE.encryptInt(integer1)
	ctxt.save("ctxt.txt")
	m = PyCtxt()
	m.load("ctxt.txt")
	if integer1 == HE.decryptInt(m):
		print("Save and Load is working correctly!")
```

What was causing the error is this assignment:

```
result = m.load("ctxt.txt")
print(HE.decryptInt(result))
```

**Comment by 767472021** [comment]
> contextGen

You can try Windows, but it's not solved.

**Comment by 767472021** [comment]
> The following code correctly saving-loading:
> 
> ```
>         HE = Pyfhel()
> 	HE.contextGen(p=65537)
> 	HE.keyGen()
> 	integer1 = 10
> 	ctxt = HE.encryptInt(integer1)
> 	ctxt.save("ctxt.txt")
> 	m = PyCtxt()
> 	m.load("ctxt.txt")
> 	if integer1 == HE.decryptInt(m):
> 		print("Save and Load is working correctly!")
> ```
> 
> What was causing the error is this assignment:
> 
> ```
> result = m.load("ctxt.txt")
> print(HE.decryptInt(result))
> ```

You can try Windows, but it's not solved.

**Comment by ibarrond** [comment]
Related to #54, solved in 73434f1 . Saving and loading works correctly in Windows and in Linux.
